content/inside-rust/sunsetting-the-rustwasm-github-org.md

```diff
@@ -0,0 +1,129 @@
+working group was quite active and helped Rust's support for WebAssembly
+flourish. Tools such as [`wasm-bindgen`] and [`wasm-pack`] were created as part
+of this effort and continue to be used to this day. After 2019 though the
+organization saw a drastic reduction in activity and most projects have been in
+maintenance mode for nearly 5 years at this point.
+
+While the working group has been officially archived for over a year now the
+repositories in the [rustwasm] organization continued on and continued to see
+some use and minor maintenance. This has perpetuated a confusing situation for
+users and maintainers alike where it's not clear what the maintenance story for
+many of these repositories are and how to move forward with the repositories
+that are actively in use.
+
+## Sunsetting the `rustwasm` Organization
+
+In September of 2025 the [rustwasm] GitHub organization will be archived to
+cease all activity within the organization. This will help clarify that
+[rustwasm] is no longer a central hub for all things Rust-and-WebAssembly but
+instead is a historical archive of efforts.
+
+The [rustwasm] organization has a number of repositories within it, most of
+which haven't seen any activity in quite some time. The current plan at this
+time is to arrange for [`wasm-bindgen`] to be transferred to a new
+[project-specific organization][wbgorg] and additionally add new fresh
+maintainers to assist with project direction going forward. All other
+repositories in the [rustwasm] organization will be archived or transferred to
+their existing maintainers if needed.
+
+### Transferring `wasm-bindgen`
+
+The [`wasm-bindgen`] project is relied on in a number of contexts today and
+existing users have expressed interest in both maintaining and growing the
+repository over time. The [rustwasm] organization is no longer a great home for
+this work so the plan is to transfer it out to a new home in a neutral location
+not owned by any one person or company. This transfer will be coupled with
+inviting new maintainers to the repository for improved project maintenance in
+addition to evolving the project.
+
+While a critical mass of new maintainers has already been reached, if you are
+interested in helping out with maintenance [an issue has been
+created][wasm-bindgen-help] to coordinate efforts around maintenance with
+[`wasm-bindgen`]. Feel free to leave a comment there to help out with this
+transition.
+
+### Archiving other repositories
+
+The current plan at this time is to archive all other repositories in the
+[rustwasm] organization. This includes repositories such as:
+
+* https://github.com/rustwasm/wasm-pack
+* https://github.com/rustwasm/gloo
+* https://github.com/rustwasm/twiggy
+* https://github.com/rustwasm/walrus
+* https://github.com/rustwasm/weedle
+* (see https://github.com/orgs/rustwasm/repositories for the full list)
+
+If you are an preexisting maintainer of one of these repositories please reach
```

Comment:
It might be worth adding a note here first that use of these dependencies which are foundational to the `wasm-bindgen` project will be refactored into the core project repo as part of the transition.

Reply:
Or, at the very least, calling out that at least wasm-pack and Walrus will be inlined in a way that is specific to wasm-bindgen without the former scope. Basically the point being to somehow note the incompatibility between this deprecation and wasm-bindgen that wasm-bindgen isn't going to have unmaintained dependencies itself.